### PR TITLE
Nootan patch basic skel

### PR DESCRIPTION
**Pull request by nootanghimire**

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+	let config = this._config;
+	if(config.baseURL === undefined || config.baseURL === null) {
+	    return Promise.reject(new Error("Check Configuration, BaseURL is not set properly!"));
+	}
+	//Also use Regex to check url in later versions
+	let _joiner;
+	if(config.baseURL.slice(-1) !== '/') {
+	    _joiner = (url[0] !== '/') ? '/' : '';
+	} else {
+	    _joiner = (_joiner === '') ? '/' : '';
```
Comment (nootanghimire) [inline app.js]: This is incorrect. I'll push a quick bugfix here.

Comment (nootanghimire) [inline app.js]: Fixed in next commit

**Comment by nootanghimire** [comment]
What's this status check?

**Comment by cyberhck** [comment]
Actually that's because no one reported github that status is passing, (we need to tell travis that this is done) for that you'd need to write some tests first! But I've disabled them for now.

**Comment by cyberhck** [comment]
See that avoid bugs by automatically running tests, stuff.

**Pull request merged**